client/state/audio/test/middleware.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy, stub } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import middleware from '../middleware';
+import { HAPPYCHAT_RECEIVE_EVENT } from 'state/action-types';
+
+describe( 'Audio Middleware', () => {
+	let next;
+	let store;
+	let play;
+
+	beforeEach( () => {
+		next = spy();
+
+		store = {
+			dispatch: spy(),
+			getState: stub(),
+			replaceReducers: spy(),
+			subscribe: spy(),
+		};
+
+		store.getState.returns( Object.create( null ) );
```

Comment:
can probably just skip all these. the tests don't even concern themselves with them.